docs/guides/bugzilla.rst

```diff
@@ -0,0 +1,208 @@
+Bugzilla Guide
+==============
+
+Contributor guidelines
+----------------------
+We have developed several tools/workflows to make the process
+for dealing with Bugzilla bugs (BZs) easier for contributors.
+Here, we discuss:
+
+* :ref:`metadata_markers` for test cases: ``blockers``, ``coverage``, and ``automates``
+* the :ref:`miq_bz_command` command
+* :ref:`bugzilla_docblock`
+
+.. _metadata_markers:
+
+Metadata Markers
+^^^^^^^^^^^^^^^^
+* **blockers**: this marker is used when a test case is `blocked` by a bugzilla bug.
+  To use this marker simply add a ``@pytest.mark.metadata(blockers=[BZ(<bug_id>])`` decorator
```

Comment:
we can also block based on a GH issue as well. But I am not sure if this belongs in here since this is only talking about bugzilla

Reply:
Yeah, I think that should be in a different portion of our Docs, then we can mention it and add a link here.